linux_os/guide/system/auditing/package_audit_installed/rule.yml

```
@@ -39,6 +39,12 @@ ocil_clause: 'the package is not installed'
 
 ocil: '{{{ ocil_package("audit") }}}'
 
+fix: |-
+    Install the audit service (if the audit service is not already installed) with the following command:
+
+    $ {{{ package_install("audit") }}}
```

This macro already contains a dollar, so you will have 2 dollars.

The generated code is:

```
fix: "Install the audit service (if the audit service is not already installed) with\
    \ the following command:\n\n$ \n$ sudo dnf install audit"
```

```suggestion
    {{{ package_install("audit") }}}
```